### DISCUSSION TITLE
App Crash While loading large DPI images in Linux

### DISCUSSION THREAD

**Discussion by nitish14s** [unoplatform/uno#18731] [Q&A] [answered]
I am trying to load high DPI images, say 600dpi or 1200 dpi images into a gridview. I have a collection of 20 images to be loaded. But while into the view, the application is crashing in linux. Is there any way I can descale the images while showing in the view ? The same code does not have any issue while showing in Windows as well WSL on Windows. By the way, attaching the model where I get the ImageSource and Binding to an Image control in GridViewItemTemplate
![Screenshot 2024-11-07 234232](https://github.com/user-attachments/assets/df6660ee-add8-4aff-ba8a-b37bfe3a2fdf)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. `ImageSource` does not yet the `DecodePixelWidth`/`DecodePixelHeight` properties that can restrict the decoding size, but you can use SkiaSharp directly to decode the image and reduce its size.

Could you tell how the app is crashing?
